ProcessMaker/Events/RequestError.php

```diff
@@ -57,4 +58,26 @@ public function getEventName(): string
     {
         return 'RequestError';
     }
+
+    /**
+     * Dispatch the event if the process request is not rate limited
+     */
+    public static function dispatchIfNotRateLimited(ProcessRequest $request, string $error)
+    {
+        $key = 'process-request-errors:' . $request->getKey();
+        $limit= config('app.process_request_errors_rate_limit', 1);
```

Comment:
code style: add an empty space before the equal sign

Reply:
Fixed